mllib/src/test/scala/org/apache/spark/mllib/clustering/GaussianMixtureSuite.scala

```diff
@@ -148,6 +148,30 @@ class GaussianMixtureSuite extends SparkFunSuite with MLlibTestSparkContext {
     }
   }
 
+  test("model prediction, parallel and local") {
+
```

Comment:
scala style: Please eliminate all of these extra newlines and only use newlines for logical divides.  See other tests for examples.

Please review [https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide]